homeassistant/components/ohme/sensor.py

```diff
@@ -61,6 +66,13 @@ class OhmeSensorDescription(OhmeEntityDescription, SensorEntityDescription):
         state_class=SensorStateClass.TOTAL_INCREASING,
         value_fn=lambda client: client.energy,
     ),
+    OhmeSensorDescription(
+        key="battery",
+        native_unit_of_measurement=PERCENTAGE,
```

Comment:
You can use translation keys to mention that this is for the connected vehicle

Reply:
Also, if the vehicle is disconnected, what does it show

Reply:
Done. When nothing is connected it'll either:
* Still show the up to date battery % - if you've linked the car with the charger in the manufacturers app,
or
* Show the amount of charge added in the last session - if no car is linked

Either way, its showing what you would get if you looked in the Ohme app, charging or not.